components/AppBar/src/MDCAppBarNavigationController.m

```diff
@@ -126,6 +126,14 @@ - (void)injectAppBarIntoViewController:(UIViewController *)viewController {
   UIScrollView *trackingScrollView =
       [self findFirstInstanceOfUIScrollViewInView:viewControllerView];
 
+  if ([self.delegate respondsToSelector:@selector
+                     (appBarNavigationController:
+                         trackingScrollViewForViewController:suggestedTrackingScrollView:)]) {
+    trackingScrollView = [self.delegate appBarNavigationController:self
```

Comment:
Why replace the default one with the nullable return value? Perhaps the client is suggesting, "I don't want to decide"?

Reply:
In this case the documented behavior is to return the provided suggestion.

Reply:
Ack.